extensions/interactions/FractionInput/directives/fraction-input-interaction.component.html

```diff
@@ -0,0 +1,24 @@
+<form autocomplete="off" [attr.disabled]="!isValid ? true : null" role="form" class="form-horizontal fraction-input-interaction">
+  <!-- This ng-form is needed for submitting answer on hitting enter. -->
+  <input type="text" name="answer" [(ngModel)]="answer" (ngModelChange)="answerValueChanged()" class="form-control" (keyup.enter)="submitAnswer()"
+         placeholder="{{customPlaceholder !== '' ? customPlaceholder : (allowNonzeroIntegerPart ? ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER' | translate) : ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER_NO_INTEGER' | translate))}}">
```

Comment:
While we are editing this file should we try to make this more readable? Double-nested inline if-else clauses are quite hard to understand.

Reply:
Agreed. Changed it.